attributes/ldap_config.rb

```diff
@@ -1,4 +1,9 @@
 default['grafana']['ldap_verbose_logging'] = false
+default['grafana']['ldap_multi_tree_auth'] = false
+# When the ldap_multi_tree_auth attribute is set to true, it is possible to configure
```

Comment:
it is not easy without a refactor, because of hierarchial structure [[server]] [attributes]